app/src/views/public/public-view.vue

```diff
@@ -3,10 +3,10 @@
 		<div class="container" :class="{ wide }">
 			<div class="title-box">
 				<div v-if="branding && branding.project_logo" class="logo" :style="{ backgroundColor: branding.project_color }">
-					<img :src="logoURL" :alt="branding.project_name || 'Logo'" />
+					<v-image :src="logoURL" :alt="branding.project_name || 'Logo'" />
 				</div>
 				<div v-else class="logo" :style="{ backgroundColor: branding.project_color }">
-					<img src="./logo-light.svg" alt="Directus" class="directus-logo" />
+					<v-image src="./logo-light.svg" alt="Directus" class="directus-logo" />
```

Comment:
Should this be reverted to `<img>`?
Alternatively, the src could be `/admin/src/views/public/logo-light.svg` 🤔 

```suggestion
					<img src="./logo-light.svg" alt="Directus" class="directus-logo" />
```

Reply:
Yes 👍🏻